templates/skeleton/package.json

```diff
@@ -29,7 +28,7 @@
     "@graphql-codegen/cli": "5.0.2",
     "@remix-run/dev": "^2.10.1",
     "@remix-run/eslint-config": "^2.10.1",
-    "@shopify/cli": "^3.64.0",
+    "@shopify/cli": "0.0.0-snapshot-20240729112727",
```

Comment:
I assume this will change to the new version of the CLI when they merge the other PR?